couchbase/admin/query/v1/query.proto

```diff
@@ -123,3 +126,13 @@ message BuildDeferredIndexesResponse {
 
   repeated Index indexes = 1;
 }
+
+message WaitForIndexOnlineRequest {
+  string bucket_name = 1;
+  string scope_name = 2;
+  string collection_name = 3;
+  string name = 4;
```

Comment:
The management APIs take a list of index names to watch. Should the `name` field here be a list, so the SDKs don't need to issue a separate request for each of potentially many indexes?

Reply:
I believe that we decided on this approach so that we could meaningful errors on a per index name basis, such as if the index doesn't exist.

Reply:
The SDKs needs to multiplex requests, this allows the server to transmit more granular details back to the client, specifically around error conditions.